ledger/sandbox/src/main/resources/db/migration/V5__Add_packages.sql

```diff
@@ -0,0 +1,31 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V5: List of packages
+--
+-- This schema version adds a table for tracking DAML-LF packages.
+-- Previously, packages were only stored in memory and needed to be specified through the CLI.
+---------------------------------------------------------------------------------------------------
+
+
+
+CREATE TABLE packages (
+  -- The unique identifier of the package (the hash of its content)
+  package_id         varchar primary key   not null,
+  -- Packages are uploaded as DAR files (i.e., in groups)
+  -- This field can be used to find out which packages were uploaded together
+  submission_id      varchar               not null,
+  -- A human readable description of the package source
+  source_description varchar,
+  -- The size of the original uploaded package, in bytes
+  size               bigint                not null,
```

Comment:
why do we need this?

Reply:
https://github.com/digital-asset/daml/blob/master/ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/admin/package_management_service.proto#L70, but thinking about it, I can replace it by `octet_length(package)`.